service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,240 @@
+               <field-map field-name="contactMechId" from="routeSegment.originTelecomContactMechId"></field-map>
+           </entity-find-one>
+           <entity-find-one entity-name="TelecomNumber" value-field="destNumber">
+               <field-map field-name="contactMechId" from="routeSegment.destTelecomContactMechId"></field-map>
+           </entity-find-one>
+
+           <!--Searching Origin and Dest of Organization -->
+           <entity-find-one entity-name="Person" value-field="originPerson">
+               <field-map field-name="partyId" from="shipment.fromPartyId"></field-map>
+           </entity-find-one>
+           <entity-find-one entity-name="Person" value-field="destPerson">
+               <field-map field-name="partyId" from="shipment.toPartyId"></field-map>
+           </entity-find-one>
+
+           <!-- Searching Country and State/Province Code -->
+           <entity-find-one entity-name="Geo" value-field="originGeoStateProvinceData">
+               <field-map field-name="geoId" from="originPostalAddress.stateProvinceGeoId"/>
+               <select-field field-name="geoCodeAlpha2"/>
+           </entity-find-one>
+           <entity-find-one entity-name="Geo" value-field="originGeoCountryData">
+               <field-map field-name="geoId" from="originPostalAddress.countryGeoId"/>
+               <select-field field-name="geoCodeAlpha2"/>
+           </entity-find-one>
+           <entity-find-one entity-name="Geo" value-field="destGeoStateProvinceData">
+               <field-map field-name="geoId" from="destPostalAddress.stateProvinceGeoId"/>
+               <select-field field-name="geoCodeAlpha2"/>
+           </entity-find-one>
+           <entity-find-one entity-name="Geo" value-field="destGeoCountryData">
+               <field-map field-name="geoId" from="destPostalAddress.countryGeoId"/>
+               <select-field field-name="geoCodeAlpha2"/>
+           </entity-find-one>
+
+           <!-- Extracting ShipmentMethod/ServiceType details-->
+           <set field="shipmentMethodEnumId" from="shipmentMethodEnumId ?: routeSegment.shipmentMethodEnumId"/>
+           <if condition="!shipmentMethodEnumId">
+               <return type="danger" message="In get#ShippingLabel Service, no shipmentMethod available."/>
+           </if>
+
+           <!-- Address details for requestMap -->
+           <set field="saddress" from="[streetLines:[originPostalAddress.address1], city:originPostalAddress.city,
+                 stateOrProvinceCode:originGeoStateProvinceData.geoCodeAlpha2, postalCode:originPostalAddress.postalCode,
+                 countryCode:originGeoCountryData.geoCodeAlpha2]" type="NewMap"/>
+           <set field="scontact" from="[personName:originPerson.firstName, phoneNumber:destNumber.contactNumber,
+                 companyName:originOrganization.organizationName]" type="NewMap"/>
+           <set field="raddress" from="[streetLines:[destPostalAddress.address1, destPostalAddress.address2],
```

Comment:
Can we use the generic service of get#AddressMap to get all address details instead of preparing multiple times?